app/models/cargo.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class Cargo < ApplicationRecord
+  validates :name,
```

Comment:
Will these validations be enough?

Reply:
Do you mean, that yet should check for character limit?

Reply:
> Do you mean, that yet should check for character limit?

Not only. With text_field in form I can put for example "New-York" as a weight, or "1337" as an email.

Reply:
now changed